src/Bicep.LangServer/Handlers/InsertResourceHandler.cs

```diff
@@ -94,7 +94,7 @@ public Task<Unit> Handle(InsertResourceParams request, CancellationToken cancell
                 {
                     // First attempt a direct GET on the resource using the Bicep type API version.
                     resource = await azResourceProvider.GetGenericResource(
-                        context.Compilation.Configuration,
+                        context.Compilation.GetEntrypointSemanticModel().Configuration,
```

Comment:
Could you create a variable named `model` at the start of this function? It wasn't immediately obvious at this point in the code why we're using the entryopint semantic model.